tests/AutoDiscoverTest.php

```diff
@@ -22,9 +22,11 @@ class AutoDiscoverTest extends BaseTestCase
 
     protected function setUp(): void
     {
-        $this->originalEnvPath = null ?? $_SERVER['CHROME_PATH'];
+        if (false !== \array_key_exists('CHROME_PATH', $_SERVER)) {
+            $this->originalEnvPath = null ?? $_SERVER['CHROME_PATH'];
```

Comment:
`null ??` doesn't do anything, and can be removed.

Reply:
Fixed.